streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientTagAwareStandbyTaskAssignor.java

```diff
@@ -211,17 +194,32 @@ static void assignStandbyTasksToClientsWithDifferentTags(final ConstrainedPriori
                 break;
             }
 
-            clientStates.get(clientOnUnusedTagDimensions).assignStandby(activeTaskId);
-
+            final ClientState clientStateOnUsedTagDimensions = clientStates.get(clientOnUnusedTagDimensions);
             countOfUsedClients++;
             numRemainingStandbys--;
 
+            log.debug("Assigning {} out of {} standby tasks for an active task [{}] with client tags {}. " +
```

Comment:
This was not request during the previous PR review, but I think it can be useful for debugging to have extra logs.